Documentation/resources.md

```diff
@@ -0,0 +1,19 @@
+## Routes
```

Comment:
I imagine that we'll need some introduction explaining the goal of this page.

```
This document explains the following resources deployed and managed by the Cluster Monitoring Operator:
* ClusterRoles
* Roles
* Routes
* Services
```

Reply:
Also, per the request in the docs Jira, I think we should add a warning about accessing API endpoints with similar info to what we include for the `/federate` endpoint: https://docs.openshift.com/container-platform/4.14/monitoring/accessing-third-party-monitoring-apis.html#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_accessing-third-party-monitoring-apis.

I'd think it would be especially important to warn about overloading the endpoint with queries.

WDYT?

Reply:
Let's add the following, and make the 4 items in the list hyperlinks to the separate sections:

*******

This document describes the following resources deployed and managed by the Cluster Monitoring Operator (CMO):

* ClusterRoles
* Roles
* Routes
* Services

Important!

In certain situations, accessing endpoints can degrade the performance and scalability of your cluster, especially if you use endpoints to retrieve, send, or query large amounts of metrics data. 

To avoid these issues, follow these recommendations:

* Avoid querying endpoints frequently. Limit queries to a maximum of one every 30 seconds.
* Do not try to retrieve all metrics data via the `/federation` endpoint. Query it only when you want to retrieve a limited, aggregated data set. For example, retrieving fewer than 1,000 samples for each request helps minimize the risk of performance degradation.
* [What else?]